test/cli/formatters/checkstyle.spec.js

```diff
@@ -44,9 +44,9 @@ describe('CLI', () => {
           }
 
           expect(stderr).to.be.equal('')
-          done()
         }
       )
+      done()
```

Comment:
This is used incorrectly! The `done()` call must appear at the end of the `exec` callback and not at the end of the `it` function.
Otherwise, the entire `exec` callback is not checked.

Please read more about it here: https://mochajs.org/#asynchronous-code

Please revert your changed to the tests